examples/go_over_there.py

```diff
@@ -70,7 +70,12 @@ def reset():
 
     for o in balls:
         if target_position is not None:
-            o.position.move_towards_ip(target_position, o.speed * delta_time)
+            try:
+                o.position.move_towards_ip(target_position, o.speed * delta_time)
+            except AttributeError:
+                raise RuntimeError(
+                    f"Version {pg.__version__} doesn't have Vector.move_towards function.\nPlease update to >=2.1.3"
```

Comment:
Just another code formatting thing, in general it's good practice to avoid long one liner strings.
This string can be broken down into smaller strings that are in different lines and python will concatenate them. This helps for better readability.